### PR TITLE
ref(xmpp): adjust last success time tracker for bosh and websocket

### PR DESCRIPTION
**Pull request by paweldomas**

### CODE REVIEW

File: modules/xmpp/StropheLastSuccess.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    /**
+     * Starts tracking requests on the given connection.
+     *
+     * @param {Object} stropheConnection - Strophe connection instance.
+     */
+    startTracking(stropheConnection) {
+        const originalRawInput = stropheConnection.rawInput;
+
+        stropheConnection.rawInput = function(...args) {
+            this._lastSuccess = new Date().getTime();
```
Comment (saghul) [inline modules/xmpp/StropheLastSuccess.js]: use performane.now() ?

Comment (paweldomas) [reply modules/xmpp/StropheLastSuccess.js]: do you know if that's available on both web and mobile RN?

Comment (saghul) [reply modules/xmpp/StropheLastSuccess.js]: We polyfill it with Date.now() IIRC because of callstats, but please double check

Comment (paweldomas) [reply modules/xmpp/StropheLastSuccess.js]: well  I dont see point in doing that to be honest

Comment (paweldomas) [reply modules/xmpp/StropheLastSuccess.js]: having it in the "since epoch" format makes more sense in this context for me

**Comment by paweldomas** [comment]
jenkins test this please

**Pull request merged**